src/app/prihlasit/page.tsx

```diff
@@ -1,8 +1,8 @@
 'use client';
```

Comment:
The user that is not logged in should not see the "zucastnit se" preferably, they should be prompted to login/register (redirect him to login page)
![CleanShot 2024-05-17 at 14 47 05](https://github.com/GLObus303/sportujspolu/assets/12488648/8ced756d-e60c-4628-8a0f-80f233b030c4)

Reply:
Notend for next PR